sympy/functions/elementary/tests/test_miscellaneous.py

```diff
@@ -394,3 +394,17 @@ def test_instantiation_evaluation():
         A, B = B, A
     assert Min(w, Max(x, y), Max(v, x, z)) == Min(
         w, Max(x, Min(y, Max(v, z))))
+
+def test_rewrite_as_Abs():
+    from itertools import permutations
+    from sympy.functions.elementary.complexes import Abs
+    from sympy.abc import x, y, z
+    def test(e):
+        free = e.free_symbols
+        a = e.rewrite(Abs)
+        for i in permutations(range(len(free))):
+            reps = dict(zip(free, i))
+            assert a.xreplace(reps) == e.xreplace(reps)
+    test(Min(x, y))
+    test(Max(x, y))
+    test(Min(x, y, z))
```

Comment:
Could you please add one more test: `test(Min(Max(w, x), Max(y, z)))` and don't forget to press return after that line and to import `w` at line 401 above.

Reply:
I added one more assertion in the test; if all passes, then this is good to go.